### ISSUE TITLE
Canceling of Welding operation should not clean editor

### ISSUE THREAD

**Issue by Alewx** [girka2k/UbioWeldContinued#34]
Currently the scene gets cleared no matter what the user decides.
In case the user finds in the last minute before saving the welded part, that something is missing or wrong, the cancel button should keep the things in place.

**Comment by girka2k** [comment]
In this regard, I have long had a question: why do we clearing the scene anyway?
For me it is in any case rather uncomfortable

**Comment by Alewx** [comment]
I have no idea, this was initial by Ubio, and might be related to memory saving but I'm not sure

**Comment by girka2k** [comment]
I propose to test the necessity of cleaning the editor 
184718c23d8f58a7ffb608d21b4133d8626a9302

**Comment by Alewx** [comment]
That is a good start, I still would prefer to keep the craft in the scene when the user cancels the saving.

**Comment by girka2k** [comment]
I made some tests without clearing of editor and found no problems.
It seems to me, we should always disable the clearing in the event of cancellation and depending on the settings - after welding, don't you think?

**Comment by girka2k** [comment]
No, all is not well. Parts are broken between pressing the "Weld it" button and a windowInfo appears. Attachment nodes are shifted.
No matter handled the whole ship or only the selected parts.
No matter, whether we kept the results of processing or not.
Thus, at the moment cleaning of the editor is required.

Two stacked FL-T200 before welding:
![2014-10-24 09-41-48 kerbal space program](https://cloud.githubusercontent.com/assets/6450733/4765620/537ef3d0-5b41-11e4-9b55-7eef1480b2ec.png)
Same parts after welding without saving
![2014-10-24 09-42-13 kerbal space program](https://cloud.githubusercontent.com/assets/6450733/4765621/5799a37a-5b41-11e4-9ef3-4ad3eaba6f9c.png)

**Comment by Alewx** [comment]
added Scene cleaning in any case in https://github.com/Alewx/unofficailUbioWeld/commit/7d44fc665e2401c16a25fbff3deed0c9f79c2a85
